library/swift/src/filters/FilterDataStatus.swift

```diff
@@ -15,13 +15,13 @@ public enum FilterDataStatus: Equatable {
   ///
   /// This filter will continue to be called with new chunks of data.
   ///
-  /// Returning `continue` from `onRequestData()`/`onResponseData()` or calling
-  /// `continueRequest()`/`continueResponse()` MUST be called when continued filter iteration is
+  /// Returning `continue` or `resumeIteration from another filter invocation or calling
+  /// `resumeRequest()`/`resumeResponse()` MUST be called when continued filter iteration is
```

Comment:
```suggestion
  /// `resumeRequest()`/`resumeResponse()` MUST occur when continued filter iteration is
```

Reply:
I still think "must be called" is accurate and maybe clearer?